### PR TITLE
[BACKEND] notifications/seen endpoints

### PR DESCRIPTION
**Pull request by umutoksuz**
## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Description

Below endpoints are implemented:
- /notifications/seen (POST)
  - Changes status of all notifications of the user to 'seen'
- /notifications/seen/{notification_id} (POST)
  - Changes a specific notification of the user to 'seen'

Also tests for these functions added.

# Test results

```
System check identified no issues (0 silenced).
...........................................................................
----------------------------------------------------------------------
Ran 75 tests in 6.210s

OK
```

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have checked my code and corrected any misspellings
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: ecommerce/backend/API/views/notification.py [hunk 1]
```diff
@@ -16,4 +16,36 @@ def manage_notifications(request):
...
+def single_notification_seen(request, notification_id):
+    user = request.user
+
+    notification = Notification.objects.filter(pk=notification_id).first()
+    if notification is None:
+        return Response(status=status.HTTP_404_NOT_FOUND)
+    
+    if notification.user != user:
+        return Response(status=status.HTTP_403_FORBIDDEN)
+
```
Comment (bekir96) [inline ecommerce/backend/API/views/notification.py]: Can you add status serializer to ease control on frontend side

**Review by bekir96 — CHANGES_REQUESTED** [review]
Status serializer may be important, please check

File: ecommerce/backend/API/views/notification.py [hunk 2]
```diff
@@ -16,4 +16,36 @@ def manage_notifications(request):
...
+    return Response({'status': {'successful': True, 'message':'All notifications of user made seen successfully.'}})
+
+@api_view(['POST'])
+@permission_classes([permissions.IsAuthenticated])
+def single_notification_seen(request, notification_id):
+    user = request.user
+
+    notification = Notification.objects.filter(pk=notification_id).first()
+    if notification is None:
+        return Response({'status': {'successful': True, 'message':'All notifications of user made seen successfully.'}}, status=status.HTTP_404_NOT_FOUND)
```
Comment (furkannane) [inline ecommerce/backend/API/views/notification.py]: Status should be changed here

**Review by furkannane — CHANGES_REQUESTED** [review]
Wrong response status should be changed

**Review by bekir96 — APPROVED** [review]
All changes seem done and you can merge, good work 👍

**Review by furkannane — APPROVED** [review]
Things seem to be fine now

**Pull request merged**